.github/workflows/integration.yml

```diff
@@ -1,6 +1,9 @@
-name: Integration
+name: CI
 
-on: [pull_request]
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
https://twitter.com/natfriedman/status/1271253144442253312

Reply:
It's about time!